includes/class-wc-pakettikauppa-shipping-method.php

```diff
@@ -118,78 +117,78 @@ public function init_form_fields() {
           /* Start new section */
           array(
             'title' => __( 'Shipping settings', 'wc-pakettikauppa' ),
-            'type' => 'title',
+            'type'  => 'title',
           ),
 
-          'active_shipping_options' => array(
-            'title'   => __( 'Active shipping options', 'wc-pakettikauppa' ),
-            'type'    => 'multiselect',
-            'options' => $this->wc_pakettikauppa_shipment->services(),
+          'active_shipping_options'    => array(
+            'title'       => __( 'Active shipping options', 'wc-pakettikauppa' ),
+            'type'        => 'multiselect',
+            'options'     => $this->wc_pakettikauppa_shipment->services(),
             'description' => __( 'Press and hold Ctrl or Cmd to select multiple shipping methods.', 'wc-pakettikauppa' ),
             'desc_tip'    => true,
           ),
 
-          'fee' => array(
+          'fee'                        => array(
```

Comment:
You are adding a lot of space in variable/array definitions. Is that intentional?

Reply:
This might be a bug of PHPCS or misconfiguration, but PHPCS attempts to align all array arrows even if not directly above/below. E.g. the arrow on line 131 is on the same level as the arrow on line 123 and 109.